packages/cozy-harvest-lib/src/components/KonnectorConfiguration/ConfigurationTab/Contracts.jsx

```diff
@@ -22,28 +31,70 @@ const makeContractsConn = ({ account, konnector }) => {
   }
 }
 
+const EditModal = ({ contract, dismissAction }) => {
+  return (
+    <Modal mobileFullscreen dismissAction={dismissAction}>
```

Comment:
I thought that we shouldn't use Modal but ExpModal now? I'm a bit lost...

Reply:
Frankly, I do not know the status of this and since it is still named ExpModal, I am not accustomed to use it. I will take a look.

Reply:
Let's keep this modal, but let's talk with @joel-costa and @y-lohse on Monday about that point :)